File: Day 23/MultiScreen_2/app/src/main/java/com/example/multiscreen_2/MainActivity.java
package com.example.multiscreen_2;

import androidx.appcompat.app.AppCompatActivity;

import android.content.Intent;
import android.os.Bundle;
import android.view.View;
import android.widget.TextView;
import android.widget.Toast;

public class MainActivity extends AppCompatActivity {

    TextView text;

    public static final String MEME = "mc";



    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);
    }

    public void age(View v)
    {
        Toast.makeText(this, "plz wait.... loading", Toast.LENGTH_SHORT).show();

        Intent i = new Intent(this,MainActivity2.class);

        text = findViewById(R.id.editTextText);

        String c = text.getText().toString();

        i.putExtra(MEME,c);

        startActivity(i);






    }

}

File: Siddhi mam Tasks/Day 38/Model passing in Intent (Arraylist)/app/src/main/java/com/example/mock_review/MainActivity.kt
package com.example.mock_review

import android.content.Intent
import android.net.Uri
import androidx.appcompat.app.AppCompatActivity
import android.os.Bundle
import android.widget.Button
import android.widget.TextView

class MainActivity : AppCompatActivity() {





    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)
        val button = findViewById<Button>(R.id.button)


        val modelList = ArrayList<MyModel>().apply {
            add(MyModel("Alice", 25))
            add(MyModel("Bob", 30))
        }

        button.setOnClickListener {
            val intent = Intent(this, MainActivity2::class.java)
            intent.putExtra("modelList", modelList)
            startActivity(intent)

        }
    }
}













